src/index.js

```diff
@@ -31,7 +32,7 @@ async function run(request, context) {
     invocation: { event },
     env: {
       CORALOGIX_API_KEY: apiKey,
-      CORALOGIX_SUBSYSTEM: subsystem,
+      CORALOGIX_SUBSYSTEM: subsystemDefault,
```

Comment:
can we name it `defaultSubsystem`? I know, I know, but we always call something that is a default, `defaultWhatever`.